tests/network/primary_pod_network.go

```diff
@@ -92,7 +92,8 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 				BeforeEach(func() {
 					var err error
 
-					vmi, err = newFedoraWithGuestAgentAndDefaultInterface(libvmi.InterfaceDeviceWithBridgeBinding())
+					const interfaceName = "default"
```

Comment:
This seems to be a "special" name, so it may be useful to just move it to the `libvmi` package as an exported const `DefaultInterfaceName`. WDYT?

Reply:
You are right